itou/templates/api/index.html

```diff
@@ -11,7 +11,7 @@
                 <div class="col-12 p-3">
                     <h1 class="h1-hero-c1">API Les emplois de l’inclusion</h1>
                     <p class="lead">
-                        L'API des <a href="{% url 'home:hp' %}" aria-label="Vers l'API des emplois de l'inclusion">emplois de l'inclusion</a> met à disposition des données de la plateforme.
+                        L'API des <a href="{% url 'home;hp' %}" aria-label="Vers les emplois de l'inclusion">emplois de l'inclusion</a> met à disposition des données de la plateforme.
```

Comment:
```suggestion
                        L'API des <a href="{% url 'home:hp' %}" aria-label="Vers les emplois de l'inclusion">emplois de l'inclusion</a> met à disposition des données de la plateforme.
```
Et je suis étonné qu'aucun test ne plante...

Reply:
#3093 pour l'absence de test